src/hotspot/share/logging/logAsyncWriter.cpp

```diff
@@ -192,10 +192,10 @@ void AsyncLogWriter::initialize() {
   AsyncLogWriter* self = new AsyncLogWriter();
   if (self->_initialized) {
     Atomic::release_store_fence(&AsyncLogWriter::_instance, self);
-    // All readers of _instance after the fence see non-NULL.
+    // All readers of _instance after the fence see non-nullptr.
```

Comment:
Nit: When "null" or "NULL" are used in prose there is some ambiguity about whether  it is referring to the general concept of nullness or the value NULL. In this case I would have written this as "see non-null". I find "non-nullptr" a bit jarring.